internal/resolution/pm/maven/job_test.go

```diff
@@ -29,9 +29,9 @@ func TestRunCmdErr(t *testing.T) {
 		},
 
 		{
-			name:  "Mvn not found",
-			error: "        |exec: \"mvn\": executable file not found in $PATH",
-			doc:   "Mvn wasn't found. Please check if it is installed and accessible by the CLI.",
+			name:  "Maven not found",
+			error: "        |exec: \"maven\": executable file not found in $PATH",
```

Comment:
I don't think we should change this line though, because the executable is called mvn. It looks like we are looking for the wrong binary otherwise

Reply:
```suggestion
			error: "        |exec: \"mvn\": executable file not found in $PATH",
```

Reply:
Totally agreed, I just meant when we refer to the name "Mvn", not the executable "mvn", sorry for the confusion!